include/logInFunctions.php

```diff
@@ -0,0 +1,71 @@
+
+<script src="/include/jquery.js"></script>
+<script type='text/javascript'>
+
+function togglePasswordVisibility(){
+	var x = document.getElementById("myInput");
+    if (x.type === "password") {
+        x.type = "text";
+    } else {
+        x.type = "password";
+    }
+}
+
+</script>
+
+
+<?php
+
+
+
+function validateFormField($formField){
+	global $Errors;
+	if(!$_REQUEST[$formField]){
+		$Errors[$formField] = 'required';
+	}
+}
+
+function createUser($username, $password){
```

Comment:
This function probably doesn't need to exist. The code inside is good, but you've already got a separate function that does all the hard stuff (`insertUser`) so you could just move the header and exit lines into the main web page.

A good way to tell if a function serves a purpose is to look at the names. `createUser` and `insertUser` mean the same thing to most people, so if they're really doing different things, they should probably have more unique names.